mpqp_scripts/setup_connections.py

```diff
@@ -53,20 +59,44 @@ def main_setup():
     import mpqp.execution.connection.aws_connection as awsc
     import mpqp.execution.connection.ibm_connection as ibmqc
     import mpqp.execution.connection.qlm_connection as qlmc
+    import mpqp.execution.connection.google_connection as cirqc
+    import mpqp.execution.connection.key_connection as keyc
     from mpqp.tools.choice_tree import AnswerNode, QuestionNode, run_choice_tree
 
+    def return_action():
+        return "", []
+
     setup_tree = QuestionNode(
         "~~~~~ MPQP REMOTE CONFIGURATION ~~~~~",
         [
-            AnswerNode("IBM configuration", ibmqc.setup_ibm_account),
-            AnswerNode("QLM configuration", qlmc.setup_qlm_account),
-            AnswerNode("Amazon Braket configuration", awsc.setup_aws_braket_account),
-            AnswerNode("Config information", print_config_info),
+            AnswerNode("IBM", ibmqc.setup_ibm_account),
+            AnswerNode("QLM", qlmc.setup_qlm_account),
+            AnswerNode("Amazon", awsc.setup_aws_braket_account),
+            AnswerNode("IonQ", keyc.config_ionq_key),
```

Comment:
I would have put IonQ and AQT under Cirq configuration, since we don't directly have access to ionq and aqt sdk/api.